src/Test/StagingStorageLoader.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbWriter\Synapse\Test;
+
+use Keboola\DbWriter\Synapse\Application;
+use RuntimeException;
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\Options\GetFileOptions;
+
+class StagingStorageLoader
```

Comment:
`StagingStorageLoader` sa pouziva v testoch, na nahratie `CSV` suboru a manifestu do `ABS`.
Nahrava sa to cez storage API do projektum ku ktoremu patri storage token, zadany v ENV.